css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: 1.25rem;
+  padding: 10px 20px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+}
+
+th:nth-child(-n+3) {
```

Comment:
Since there are only 3 `<th>` elements and we want them to all be black using `nth-child` here is a bit overkill, but good job making it work!